hawkular-command-gateway/hawkular-command-gateway-war/pom.xml

```diff
@@ -89,6 +89,11 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-api</artifactId>
```

Comment:
is the other inventory dep not sufficient? (inventory-paths). I am guessing it is not. We should ask Lukas about splitting out another inventory library if we need it.